__tests__/test.test.js

```diff
@@ -113,5 +113,48 @@ describe('GET /api/articles/:article_id', () => {
         expect(body.msg).toEqual('Bad request')
         })
     })
-    })
+ })
 
+describe('GET /api/articles', () => {
+    test('Status code 200 returned', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        })
+
+    test('Request responds with an article that has the correct properties and comment count', () => {
+        return request(app)
+        .get('/api/articles')
+        .then(({ body }) => {
+            expect(body.articles[6]).toMatchObject({
+            author: 'butter_bridge',
+            title: 'Living in the shadow of a great man',
+            article_id: 1,
+            topic: 'mitch',
+            created_at: '2020-07-09T20:11:00.000Z',
+            votes: 100,
+            article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700',
+            comment_count: 11   
+            })
+        })
+    })
+    test('Request responds with articles sorted by date in descending order', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({ body: { articles } }) => {
+            expect(articles.length).toBe(13)
+            let sorted = true
+            let date = articles[0].created_at
+            articles.forEach(article => {
+                const nextDate = article.created_at
+                if(nextDate > date) {
+                isSorted = false
+                }
+                date = nextDate
+                })
+            expect(sorted).toBe(true)
```

Comment:
Have a look at jest-sorted which will make testing for the order of things a lot easier.

https://www.npmjs.com/package/jest-sorted